translations/it.yaml

```diff
@@ -0,0 +1,347 @@
+        6702: Licenza edilizia rilasciata (con validità giuridica)
+        6703: in corso
+        6704: Concluso
+        6706: Progetto rinviato
+        6707: Domanda di costruzione respinta (con validità giuridica)
+        6708: Non realizzato
+        6709: Ritirato
+
+    building:
+      statusOptions:
+        1001: In progetto
+        1002: Autorizzato
+        1003: In costruzione
+        1004: esistente
+        1005: Non utilizzabile
+        1007: Demolito
+        1008: Non realizzato
+
+    dwelling:
+      statusOptions:
+        3001: in progetto
+        3002: Autorizzato
+        3003: In costruzione
+        3004: esistente
+        3005: Non utilizzabile
+        3007: Abrogato
+        3008: Non realizzato
+
+    statusErrorBuilding: "Edificio {buildingId}"
+    statusErrorDwelling: "Abitazione {dwellingId} nell'edificio {buildingId}"
+    statusError: "deve avere lo stato {states}."
+
+    states:
+      6701: in progetto
+      6702: autorizzato
+      6703: in costruzione
+      6704: concluso
+      6706: rinviato
+      6707: rifiutato
+      6708: non realizzato
+      6709: ritirato
+
+      1001: in progetto
+      1002: autorizzato
+      1003: in costruzione
+      1004: esistente
+      1005: Non utilizzabile
+      1007: Demolito
+      1008: non realizzato
+
+      3001: in progetto
+      3002: autorizzato
+      3003: in costruzione
+      3004: esistente
+      3005: Non utilizzabile
+      3007: Abrogato
+      3008: non realizzato
+
+  components:
+    search:
+      loading: Carica...
+      noResults: Non abbiamo trovato alcun risultato per i criteri di ricerca selezionati.
+      validation:
+        realestateIdentification: Per la ricerca avanzata occorre indicare il numero del fondo.
+        bothDatesSet: Occorre indicare entrambe le date.
+
+      realestateIdentificationHeading: Identificatore dei fondi
+      realestateIdentification:
+        EGRID: EGRID
+        number: Numero del fondo (numero della particella)
+        subDistrict: Numero del circondario del registro fondiario
+
+      createDateHeading: Periodo di realizzazione
+      createDate:
+        dateFrom: "da"
+        dateTo: "a"
+
+      modifyDateHeading: Periodo di elaborazione
+      modifyDate:
+        dateFrom: "da"
+        dateTo: "a"
+
+      extendedSearch:
+        extend: Visualizza filtri avanzati
+        collapse: Nascondi filtri avanzati
+
+    loginModal:
+      header: Effettua login nel REA
+      login: Login
+      password: Password...
+      username: Nome utente...
+      municipality: Seleziona un comune...
+      municipalitySearch: Cerca comune...
+      municipalityNoneFound: Non è stato trovato alcun comune
+      loginInfo: "<b>{user}</b> login effettuato per <b>{municipality}</b>"
+      loginIssueContact: "<b>Se non è possibile effettuare il login contattare direttamente...</b><br><br>hotline<br>housing-stat@bfs.admin.ch<br>0800 866 600<br>Dal lunedì al venerdì<br>08.30-11.30 e 14.00-16.00"
```

Comment:
Does the BfS have language specific telephone numbers?

Reply:
Seems to be the same number in italian: https://www.housing-stat.ch/it/index.html